code/game/objects/items/devices/oxycandle.dm

```diff
@@ -34,11 +36,14 @@
 	if(on)
 		var/turf/pos = get_turf(src)
 		if(volume <= 0 || istype(pos, /turf/simulated/floor/beach/water) || istype(pos, /turf/unsimulated/beach/water))
-			new/obj/item/trash/candle(src.loc) // Placeholder for new sprites
-			if(istype(src.loc, /mob))
-				src.dropped()
 			STOP_PROCESSING(SSprocessing, src)
-			qdel(src)
+			icon_state = "oxycandle_burnt"
+			item_state = icon_state
+			set_light(0)
+			update_held_icon()
+			name = "burnt oxygen candle"
+			desc = "A steel tube with the words 'OXYGEN - PULL CORD TO IGNITE' stamped on the side. A small label warns against using the device underwater. This tube exhausted its chemicals."
```

Comment:
You could just do `desc += " This tube has exhausted its chemicals."` instead. Needs grammar too.